tools/src/main/java/org/apache/rocketmq/tools/command/export/ExportConfigsCommand.java

```diff
@@ -79,10 +81,10 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook)
             Map<String, Properties> brokerConfigs = new HashMap<>();
             Map<String, List<String>> masterAndSlaveMap
                 = CommandUtil.fetchMasterAndSlaveDistinguish(defaultMQAdminExt, clusterName);
-            for (String masterAddr : masterAndSlaveMap.keySet()) {
-                Properties masterProperties = defaultMQAdminExt.getBrokerConfig(masterAddr);
+            for (Entry<String, List<String>> masterAddrEntry : masterAndSlaveMap.entrySet()) {
```

Comment:
masterAddrEntry could be renamed to masterAndSlaveEntry

Reply:
already edited